task/src/lastpencil/PencilGame.java

```diff
@@ -3,4 +3,113 @@
 import java.util.Random;
 import java.util.Scanner;
 
-//Class for Pencil Game
\ No newline at end of file
+public class PencilGame {
+    private static final Scanner scanner = new Scanner(System.in);
```

Comment:
This is all nice and good, however when is this scanner closed?

Perhaps in the context of this game this is not that relevant ("hey.. after the game the application is closed anyway, right?"). Nonetheless the scanner should be closed at some point to ensure things are closed off nicely and politely.

Can you think of a way to invoke the close()-method on the scanner-object at the end of the game?